builds/fabric8-che/plugins/keycloak-plugin-shared/src/main/java/com/redhat/che/keycloak/shared/KeycloakSettings.java

```diff
@@ -0,0 +1,48 @@
+package com.redhat.che.keycloak.shared;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.Map;
+
+import java.util.logging.Logger;
```

Comment:
well, strange yes. Maybe it would work now, but when I first worked on this, it was producing an error. Do you really want me to change / test this now, or couldn't we keep this for another issue ?